docs/Changelog.md

```diff
@@ -11004,6 +11087,61 @@ This version of the operator has been available since version 11 of the default
 <dd>tensor of bool, which should be a scalar.</dd>
 </dl>
 
+### <a name="OneHot-11"></a>**OneHot-11**</a>
+
+  Produces a one-hot tensor based on inputs.
+      The locations represented by the index values in the 'indices' input tensor will have 'on_value'
+      and the other locations will have 'off_value' in the output tensor, where 'on_value' and 'off_value'
+      are specified as part of required input argument 'values', which is a two-element tensor of format
+      [off_value, on_value]. The rank of the output tensor will be one greater than the rank of the
+      input tensor. The additional dimension is for one-hot representation. The additional dimension will
+      be inserted at the position specified by 'axis'. If 'axis' is not specified then then additional
+      dimension will be inserted as the innermost dimension, i.e. axis=-1. The size of the additional
+      dimension is specified by required scalar input 'depth'. The type of the output tensor is the same
+      as the type of the 'values' input. Any entries in the 'indices' input tensor with values outside
+      the range [-depth, depth-1] will result in one-hot representation with all 'off_value' values in the
+      output tensor.
```

Comment:
Please add equations such as
```
output[i, j, k, input[i, j, k]] = 1 for all i, j, k and 0 otherwise.
```
if `axis = -1`, and
```
output[input[i, j, k], i, j, k] = 1 for all i, j, k and 0 otherwise.
```
when `axis=0`.

Reply:
Done